### ISSUE TITLE
Wall Street Journal Not Working

### ISSUE THREAD

**Issue by TheHungryGeek** [iamadamdev/bypass-paywalls-chrome#360]
As above, please have a look

**Comment by hamorphis** [comment]
Yep. On the code below ".close-btn" cannot be found on the page

```
if (window.location.href.indexOf("wsj.com") !== -1) {
    if (location.href.includes('/articles/')) {
        document.querySelector('.close-btn').click();
    }
}

```

**Comment by hamorphis** [comment]
The weird thing is that not only the close button cannot be found, but the article is only with its first 2 paragraphs, which could mean something went wrong with cleaning the cookies.

**Comment by hamorphis** [comment]
If you add it to the use_google_bot it starts working. But I do not have rights to create a merge request.

**Comment by magnolia1234** [comment]
@hamorphis, with your fork you could make a commit and pull request?

**Comment by MPlatz1** [comment]
Stopped working yesterday

**Comment by hamorphis** [comment]
Thanks. I will have this in mind for the future.

**Comment by jlukic** [comment]
Here's the [zip of the fork](https://github.com/magnolia1234/bypass-paywalls-chrome/archive/master.zip) for those waiting on a merge

**Comment by iamadamdev** [comment]
Merged in v1.5.6
